fal/views/teams.py

```diff
@@ -135,17 +131,14 @@ def team_stats(season_str: str = season_str, year: int = year, filename: str = "
                 f.write(f"{i} - {anime}: {count} ({active_count})\n")
 
 
-def team_dist(season_str: str = season_str, year: int = year, filename: str = "lists/team_dist.txt", prep: bool = True) -> None:
+def team_dist(season_str: str = season_str, year: int = year, filename: str = "lists/team_dist.txt") -> None:
     """
     Creates a statistic of the team distribution (how many people and who chose the same team)
     This function can also be used during the game to obtain the team distribution of the current week.
     @param prep during the real game (False) or before the game (True)
     """
     week: int = config.getint('weekly info', 'current-week')
-    if not prep:
-        # Add week to the given filename
-        filename = filename[:-4] + f"_{week}" + filename[-4:]
-        raise NotImplementedError("Haven't implemented dist during real game")
+    add_week_to_filename(filename, week)
```

Comment:
this function doesn't seem to mutate `week`, it just returns a string that you don't store

Reply:
Shoot, you're right.